src/test/java/seedu/address/logic/parser/ParserUtilTest.java

```diff
@@ -203,18 +203,18 @@ public void parseEmail_validValueWithWhitespace_returnsTrimmedEmail() throws Exc
     }
     @Test
     public void parseMedicalHistory_null_throwsNullPointerException() {
-        assertThrows(NullPointerException.class, () -> ParserUtil.parseMedicalHistories(null));
+        assertThrows(NullPointerException.class, () -> ParserUtil.parseMedicals(null));
```

Comment:
It's good that you adapted the test cases for `parseMedicals`, but I think `parseMedicals` already has its own test cases from `Line 313` onwards. Have you checked if there are overlaps in test cases?